modules/sitemaps/sitemaps.php

```diff
@@ -202,6 +202,9 @@ function jetpack_load_xsl( $type = '' ) {
 
 	$transient_xsl = empty( $type ) ? 'jetpack_sitemap_xsl' : "jetpack_{$type}_sitemap_xsl";
 
+	// Add current language as suffix if WPML is active
+	$transient_xsl = $transient_xsl . wpml_transient_language();
```

Comment:
Ok, is jetpack_wpml_transient_language() good or to remove wpml completely from equation? :)